packages/components/src/FormField/FormField.css

```diff
@@ -99,7 +99,16 @@ select.formField {
 .inline {
   display: inline-block;
   width: auto;
-  vertical-align: -moz-middle-with-baseline;
+  vertical-align: baseline;
+}
+
+.inline > input,
+.inline > select {
+  display: inline-block;
+}
+
+.inline.inlineTextArea {
+  vertical-align: middle;
```

Comment:
I don't think we need to support inline TextAreas. We should perhaps in a followup officially remove support for them. I really can't think of a case where we would want this behaviour. 🤔

@chris-at-jobber Any thoughts on this part?

Reply:
I'm good with dropping support for inline TextArea 👍

Reply:
Added a new commit that removed this extra styling